docs/build/hosted-lotus-node.md

```diff
@@ -0,0 +1,643 @@
+---
+title: Hosted Lotus Node
+description: As the fundamental component of Filecoin Network, lotus node offers abundant features to developers via command-line as well as RPC API methods. 
+breadcrumb: 'Lotus Node'
+---
+
+# {{ $frontmatter.title }}
+
+{{ $frontmatter.description }} 
+
+In the Filecoin ecosystem, there are various node solutions to connect to the Filecoin network and provide  functions via RPC API calls. Developers can either install and run their own lotus node locally to interact with the Filecoin network, or connect to other lotus nodes hosted by Glif or Infura.
+
+Developers can also use the API client libraries to utilize the features of Filecoin Network since those libraries take care of the low-level logic of connecting to lotus node, requesting and handing the response via simple calls, etc. Currently, there are many libraries available:
+
++ [filecoin.js](https://filecoin-shipyard.github.io/filecoin.js/)
++ [filecoin-js-signer](https://github.com/blitslabs/filecoin-js-signer)
++ [Filecoin Signing Tools](https://github.com/Zondax/filecoin-signing-tools)
+
+In this tutorial, we will run a  **local lotus node** and use **filecoin.js** library to build examples to walk you through: 
+
+- Setting up local lotus node for Calibration TestNet
+- Using filecon.js to write code to interact with Filecoin Network to
+   - query basic blockchain info
+   - manage lotus wallet
+   - manage light wallet
+   - make a storage deal
+
+## **Pre-requirements**
+
+Before getting started, you should be familiar with:
+
++ [How Filecoin works](http://localhost:8080/about-filecoin/how-filecoin-works/)
++ [How to install a lotus node for the Calibration network](https://lotu.sh/lotus/install/prerequisites/#supported-platforms)
++ [How to use Lotus command line](https://lotu.sh/lotus/manage/lotus-cli/)
+
+Also, make sure you have all the dependencies installed:
+
++ Fully Synced Calibration TestNet lotus node
++ node.js
++ npm
+
+## Setup
+
+Before you start writing code to interact with hosted lotus node, you should make sure everything is set up properly. Especially ensure you have a local lotus node fully synced for Calibration TestNet. 
+
+### Set up lotus node
+
+If you have not installed a lotus node locally and fully synced for Calibration TestNet, you should go back and do it following the guild to [Install and launch a Lotus node](https://lotus.filecoin.io/docs/set-up/install/).
+
+To sum up, you need to:
+
+1. **Download lotus source code for the latest release**
```

Comment:
Don't bold list items. https://docs.filecoin.io/community/contribute/grammar-formatting-and-style/#bold-text